python/sdk/docs/index.rst

```diff
@@ -19,4 +19,5 @@ Resources:
 
    self
    tutorial
+   api/index
```

Comment:
That means we need to build the "main" docs first? If so this won't work when building from RTD, will it?

Reply:
My bad, that shouldn't be there